pkg/apiserver/apiserver.go

```diff
@@ -174,27 +177,27 @@ func (s *APIServer) buildHandlerChain() {
 		GrouplessAPIPrefixes: sets.NewString("api", "kapi"),
 	}
 
-	failed := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		w.WriteHeader(http.StatusUnauthorized)
-	})
-
 	handler := s.Server.Handler
+
 	handler = filters.WithKubeAPIServer(handler, s.KubernetesClient.Config(), &errorResponder{})
 	handler = filters.WithMultipleClusterDispatcher(handler, dispatch.DefaultClusterDispatch)
-	handler = filters.WithAuthorization(handler, authorizerfactory.NewAlwaysAllowAuthorizer())
 
-	authn := union.New(&authenticationrequest.AnonymousAuthenticator{}, bearertoken.New(jwttoken.NewTokenAuthenticator(s.CacheClient, s.AuthenticateOptions.JwtSecret)))
-	handler = filters.WithAuthentication(handler, authn, failed)
-	handler = filters.WithRequestInfo(handler, requestInfoResolver)
+	excludedPaths := []string{"/oauth/authorize", "/oauth/token"}
```

Comment:
It's not a good a idea to hardcode excluded paths here, we need a better solution

Reply:
These two API paths are fixed, if we need dynamic configuration, we need to add  some configurations like caddy.